python/cudf/cudf/core/series.py

```diff
@@ -3639,19 +3639,31 @@ def corr(self, other, method="pearson", min_periods=None):
         >>> import cudf
         >>> ser1 = cudf.Series([0.9, 0.13, 0.62])
         >>> ser2 = cudf.Series([0.12, 0.26, 0.51])
-        >>> ser1.corr(ser2)
+        >>> ser1.corr(ser2, method="pearson")
         -0.20454263717316112
+        >>> ser1.corr(ser2, method="spearman")
+        -0.5
         """
 
-        assert method in ("pearson",) and min_periods in (None,)
-
+       
         if self.empty or other.empty:
             return cudf.utils.dtypes._get_nan_for_dtype(self.dtype)
-
-        lhs = self.nans_to_nulls().dropna()
-        rhs = other.nans_to_nulls().dropna()
-        lhs, rhs = _align_indices([lhs, rhs], how="inner")
-
+        
+        if method == "pearson":
+            
+            lhs = self.nans_to_nulls().dropna()
+            rhs = other.nans_to_nulls().dropna()
+            lhs, rhs = _align_indices([lhs, rhs], how="inner")
```

Comment:
Can we consolidate this redundant line to just before the return statement?

Reply:
done